app/src/main/res/value-it/strings.xml

```diff
@@ -0,0 +1,277 @@
+    <string name="feedback_rating_neutral">Neutro</string>
+    <string name="feedback_rating_negative">Negativo</string>
+    <string name="feedback_error_sending">Errore nell'invio del tuo feedback</string>
+
+    <string name="navigation_item_map">Mappa</string>
+    <string name="navigation_item_favorites">Preferiti</string>
+    <string name="navigation_item_messages">Messaggi</string>
+    <string name="navigation_item_account">Account</string>
+    <string name="navigation_item_settings">Impostazioni</string>
+    <string name="navigation_item_about">Informazioni</string>
+
+    <string name="title_fragment_favorites">Preferiti</string>
+    <string name="title_fragment_messages">Messaggi</string>
+    <string name="title_fragment_settings">Impostazioni</string>
+    <string name="title_fragment_users">Utenti</string>
+    <string name="title_fragment_filter_list">Filtro Utenti</string>
+
+    <string name="message_thread_create_failed">Impossibile creare un nuovo thread di messaggi.</string>
+    <string name="messages_reload_failed">Impossibile ricaricare i messaggi.</string>
+
+    <string name="message_new_hint">Scrivi un messaggio</string>
+    <string name="message_mark_unread">Segna come non letto</string>
+    <string name="message_mark_read">Segna come letto</string>
+    <string name="message_send_failed">Impossibile inviare il messaggio.</string>
+
+    <string name="remember_password">Memorizza la password?</string>
+    <string name="button_text_forgot_password">Password dimenticata?</string>
+    <string name="url_website_base">https://www.warmshowers.org</string>
+
+    <string name="content_description_your_avatar">Il tuo avatar</string>
+    <string name="content_description_avatar_of_var">%s’s avatar</string>
+    <string name="content_description_notification_count">Conteggio delle notifiche</string>
+    <string name="content_description_members_marker_map">Membri</string>
+    <string name="content_description_member_marker_map">Membro</string>
+
+    <string name="menu_goto_profile">Mostra il profilo</string>
+    <string name="menu_goto_profiles">Mostra i profili</string>
+
+    <string name="enabled">Abilitato</string>
+
+    <string name="unknown_location">Posizione sconosciuta</string>
+
+    <string name="button_clear_filters">Cancella filtri</string>
+    <string name="filter_last_access">Ultimo accesso</string>
+    <string name="filter_diff_today">Oggi</string>
+    <string name="filter_diff_this_week">Questa settimana</string>
+    <string name="filter_diff_this_month">Questo mese</string>
+    <string name="filter_diff_this_year">Questo anno</string>
+    <string name="filter_diff_any">In qualsiasi momento</string>
+    <string name="filter_diff_short_today">1d</string>
+    <string name="filter_diff_short_this_week">1w</string>
+    <string name="filter_diff_short_this_month">1m</string>
+    <string name="filter_diff_short_this_year">1y</string>
+    <string name="filter_diff_short_any">&gt; 1y</string>
```

Comment:
Just wondering: Are those the correct Italian abbreviations for day, week, month and year?

Reply:
Yes, we usually use english abbreviation

Reply:
OK, great.